src/fidesops/migrations/versions/55d61eb8ed12_add_default_policies.py

```diff
@@ -276,7 +276,7 @@ def autogenerate_access_policy(
             "target_id": rule_target_id,
             "target_name": "Fidesops Autogenerated Access Target",
             "target_key": "fidesops_autogenerated_access_target",
-            "data_category": DataCategory("user.provided.identifiable").value,
+            "data_category": DataCategory("user").value,
```

Comment:
Should we be changing legacy migrations? This ought to be handle in the new migration that's been added. I'm not sure it actually makes a difference here, but technically speaking the migration history should be preserved.

Reply:
@seanpreston this could be changed to just a string, "user.provided.identified", but the current version of fideslang will break if we try to use a data category that doesn't exist

I think we made that change elsewhere too and it seemed to work